src/DynamoCore/DynamoCore.csproj

```diff
@@ -315,11 +325,13 @@ IF NOT EXIST "$(OutputPath)\dll" "$(OutputPath)\InstallASMForDynamo.exe" /SILENT
 copy /Y "$(SolutionDir)..\extern\LibG\x64\Release\*" "$(OutputPath)\dll"
 </PostBuildEvent>
     <PostBuildEvent>xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\*" "$(OutputPath)"
-xcopy  /Y "$(SolutionDir)..\doc\distrib\License.rtf" "$(OutputPath)"
+xcopy /Y "$(SolutionDir)..\extern\ProtoGeometry\ProtoGeometry.xml" "$(OutputPath)\$(UICulture)"
+xcopy  /Y "$(SolutionDir)..\doc\distrib\License.rtf" "$(OutputPath)\$(UICulture)"
```

Comment:
Yes it has to be localized, it gets shown on the UI when user goes to `Help > About` for the About Box. Localization team will extract it out of the MSI for localization (which is why it needs to go to `UICulture` folder).